docs/UserGuide.md

```diff
@@ -1,8 +1,6 @@
 ---
-layout: page
-title: User Guide
+USER GUIDE FOR WIFE
 ---
-
 WIFE is always right. Our product Well Informed Fridge Environment &lt;WIFE/&gt; helps users to manage 
```

Comment:
Do we want to change this out to Enhance life with WIFE?

Reply:
Will rectify after merging.